.github/workflows/build-latest.yml

```
@@ -32,9 +32,9 @@ jobs:
       sha: ${{ inputs.sha }}
       arch: amd64
       tag: ${{ inputs.tag-prefix }}latest-amd64
-      protocol_version_default: 21
-      xdr_ref: v21.1.0
-      core_ref: v21.3.1
+      protocol_version_default: 22
```

Until all the services are updated to 22, and until the network is on 22, we should leave this on 21 so local runs of the instance use 21 too.
```suggestion
      protocol_version_default: 21
```